packages/cli/src/ui/hooks/useTerminalSize.ts

```diff
@@ -6,18 +6,16 @@
 
 import { useEffect, useState } from 'react';
 
-const TERMINAL_PADDING_X = 8;
```

Comment:
reverted this padding as it wasn't needed. it was from when we didn't refesh on resize so a single Ink rendering artifact would permanently corrupt. now that isn't the case.